mapper.go

```diff
@@ -256,6 +295,93 @@ func mapRowSerial(
 	return row.Scan(destVals...)
 }
 
+// mapOuterJoinRowSerial maps rows to **Model. NULL value in ColumnSplitter column sets *Model = nil.
+// It can map inner join rows as well but using mapRowSerial is faster because less reflection logics.
+func mapOuterJoinRowSerial(
```

Comment:
I think both * and ** should be acceptable for inner/outer join in either case. There seems to be large part of duplicated codes between inner and outer. Could you migrate them into single function (within mapRowSerial)?

Reply:
Updated. Now mapRowSerial accepts * and **, and I updated example, too.